smart-app-banner.css

```diff
@@ -1,294 +1,267 @@
 .smartbanner-show {
-	margin-top: 80px;
+  margin-top: 80px;
 }
-.smartbanner-show .smartbanner {
-	display: block;
-}
-
 
-/** Default **/
 .smartbanner {
-	position: absolute;
-	left: 0;
-	top: 0;
-	display: none;
-	width: 100%;
-	height: 80px;
-	line-height: 80px;
-	font-family: 'Helvetica Neue', sans-serif;
-	background: #f4f4f4;
-	z-index: 9998;
-	-webkit-font-smoothing: antialiased;
-	overflow: hidden;
-	-webkit-text-size-adjust: none;
+  position: absolute;
+  left: 0;
+  top: 0;
+  display: none;
+  width: 100%;
+  height: 80px;
+  line-height: 80px;
+  font-family: "Helvetica Neue", sans-serif;
+  background: #f4f4f4;
+  z-index: 9998;
+  -webkit-font-smoothing: antialiased;
+  overflow: hidden;
+  -webkit-text-size-adjust: none;
+}
+
+.smartbanner-show .smartbanner {
+  display: block;
 }
 
 .smartbanner-container {
-	margin: 0 auto;
-	white-space: nowrap;
+  margin: 0 auto;
+  white-space: nowrap;
 }
 
 .smartbanner-close {
-	display: inline-block;
-	vertical-align: middle;
-	margin: 0 5px 0 5px;
-	font-family: 'ArialRoundedMTBold', Arial;
-	font-size: 20px;
-	text-align: center;
-	color: #888;
-	text-decoration: none;
-	border: 0;
-	border-radius: 14px;
-	-webkit-font-smoothing: subpixel-antialiased;
+  display: inline-block;
+  vertical-align: middle;
+  font-family: 'ArialRoundedMTBold', Arial;
+  margin: 0 5px 0 5px;
+  font-size: 20px;
+  text-align: center;
+  color: #888;
+  text-decoration: none;
+  border: 0;
+  border-radius: 14px;
+  -webkit-font-smoothing: subpixel-antialiased;
 }
 
 .smartbanner-close:active,
 .smartbanner-close:hover {
-	color: #aaa;
+  color: #aaa;
 }
 
 .smartbanner-icon {
-	display: inline-block;
-	vertical-align: middle;
-	width: 57px;
-	height: 57px;
-	margin-right: 12px;
-	background-size: cover;
-	border-radius: 10px;
+  display: inline-block;
+  vertical-align: middle;
+  width: 57px;
+  height: 57px;
+  margin-right: 12px;
+  background-size: cover;
+  border-radius: 10px;
 }
 
 .smartbanner-info {
-	display: inline-block;
-	vertical-align: middle;
-	width: 44%;
-	font-size: 11px;
-	line-height: 1.2em;
-	font-weight: bold;
+  display: inline-block;
```

Comment:
Tabs are replaced to 2-space style - not a good idea, considering that the project uses tabs everywhere else